services/metadata_service/api/tag.py

```diff
@@ -0,0 +1,139 @@
+from services.data import TaskRow
+from services.data.db_utils import DBResponse
+from services.data.postgres_async_db import AsyncPostgresDB
+from services.metadata_service.api.utils import format_response, \
+    handle_exceptions
+import json
+
+import asyncio
+
+
+class TagApi(object):
+    lock = asyncio.Lock()
```

Comment:
seems to be unused.

Reply:
I copied from other APIs but let me check if there was a purpose behind it.